timelock-impl/src/main/java/com/palantir/atlasdb/timelock/TimelockNamespaces.java

```diff
@@ -76,11 +75,16 @@ public TimelockNamespaces(
     }
 
     public TimeLockServices get(String namespace) {
+        Preconditions.checkArgument(
```

Comment:
It must go after we computeIfAbsent, to avoid race conditions where this passes, we disable and call `getIgnoring` and then this calls computeIfAbsent